### ISSUE TITLE
Error in locally running the documentation

### ISSUE THREAD

**Issue by asifshiraz** [blockstack/docs#1175]
I was thinking of improving the documentation a little here and there, so cloned the repo to run it locally.

Following all the steps work, and loads the main page and some sub-pages. But on certain others, it gives this error:

![image](https://user-images.githubusercontent.com/7054246/120375106-0195f600-c2e0-11eb-80a9-c97855b80cfb.png)

![image](https://user-images.githubusercontent.com/7054246/120375164-15415c80-c2e0-11eb-85ba-91222c6ee688.png)

TypeError [ERR_INVALID_ARG_TYPE]: The "data" argument must be of type string or an instance of Buffer, TypedArray, or DataView. Received undefined
    at Hash.update (internal/crypto/hash.js:84:11)
    at generateHash (C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:590:35)
    at C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:566:26
    at _catch (C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:546:16)
    at C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:565:16
    at readGrammarFromPath (C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:583:6)
    at _temp3 (C:\code\docs\node_modules\unified-vscode\dist\unified-vscode.cjs.development.js:639:51) {
  code: 'ERR_INVALID_ARG_TYPE'

Attn: @aulneau (Please :))

**Comment by pgray-hiro** [comment]
Hi @asifshiraz, can you provide more information? What version of `node` are you using?

**Comment by asifshiraz** [comment]
Node v14.16.1

**Comment by pgray-hiro** [comment]
It looks like you're using Windows, I tried pulling the repo on my Windows laptop and building it using Node 14.16.1. After navigating around a few pages on the local build, I'm not able to reproduce this crash. Can you provide more of the terminal output from the crash?  Here were the steps I took:

(with node 14.16.1 installed)
1. Install yarn (`npm i -g yarn`)
2. Clone docs repo (`git clone https://github.com/blockstack/docs && cd docs`)
3. Install dependencies (`yarn`)
4. Start development server (`yarn dev`)

Any more info you can provide about your process would be helpful so we can track down this issue.

**Comment by asifshiraz** [comment]
I have found the issue.

During code execution, a dependency library function readGrammarFromPath is being called, which is trying to read a file using following function:

`function getFilename(path) {
  var filename = path.split('mars/')[1];
  return filename;
}`

On my windows system, where I cloned the repo, the path to the file is 
C:\code\docs\node_modules\unified-vscode\dist\data\extraGrammars\stylus.json
But the function is assuming presence of hard coded "mars/" value in the path, and so it is returning undefined, causing the rest of the logic to break.

The specific location of this function is line 602 in \node_modules\unified-vscode\dist\unified-vscode.cjs.development.js 

I tried to follow where the stacks code is calling this unified-vscode library. Seems like its referenced from the rehype-plugin in from /common/data/mdx.ts

The error occurs for me when I click on this link http://localhost:3000/en-US/understand-stacks/testnet
